mesonbuild/compilers/compilers.py

```diff
@@ -850,6 +850,9 @@ def build_unix_rpath_args(self, build_dir, from_dir, rpath_paths, build_rpath, i
             args += ['-Wl,-rpath-link,' + lpaths]
         return args
 
+    def has_func_attribute(self, name, env):
+        raise EnvironmentException('Language %s does not support function attributes.' % self.get_display_language())
```

Comment:
Eh, you use '{}' above. '%' is better avoided.

Reply:
That's copy and pasted, I can change it.